pysat/instruments/__init__.py

```diff
@@ -16,4 +16,5 @@
            'pysat_testing', 'pysat_testing_xarray', 'pysat_testing2d',
            'pysat_testing2d_xarray', 'pysat_testmodel']
 
-from . import *
+for inst in __all__:
+    exec("from pysat.instruments import {x}".format(x=inst))
```

Comment:
Just curious, why this change?

Reply:
Better maintainability.  I don't want to write out the full list twice in case of typos.  Particularly with the instrument lists changing down the line.  Open to suggestions for cleaner implementation / better style.

Reply:
I also thought that `import *` was one of the things to eliminate based on the issue.

Reply:
The `exec` caught my attention, it's a shame it can't just be `from pysat.instruments import inst`.  But I agree with all the things you said and am not opposed to the `exec` as it is cleaner than the other options I can think of.

Reply:
OK.  I'll keep thinking about this as well, but will merge these in.